packages/netlify-cms-widget-object/src/ObjectControl.js

```diff
@@ -46,16 +54,59 @@ export default class ObjectControl extends Component {
     return true;
   }
 
+  processControlRef = (field, wrappedControl) => {
+    if (!wrappedControl) return;
+    const name = field.get('name');
+    const list = field.get('widget') == 'list';
+    const object = field.get('widget') == 'object';
+    if (list) {
+      this.componentValidate[name] = wrappedControl.innerWrappedControl.validateList;
+    } else if (object) {
+      this.componentValidate[name] = wrappedControl.innerWrappedControl.validateObject;
+    } else {
+      this.componentValidate[name] = wrappedControl.validate;
+    }
+    this.objectUniqueFieldIds.push(wrappedControl.props.uniqueFieldId);
+  };
+
+  validateObject = () => {
```

Comment:
See first suggestion for `EditorControlPane.js`

```suggestion
  validate = () => {
```